bot/scripts/wiki_to_markdown.py

```diff
@@ -50,6 +50,9 @@ def wiki_to_markdown(text, wiki_url_prefix='https://www.weezerpedia.com/wiki/'):
     # Trim any leading or trailing whitespaces
     text = text.strip()
 
+    url_pattern = r'http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\\(\\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
```

Comment:
the URLs just take up character limit and dont add value to response since gpt cannot traverse them

Reply:
thinking on this more. i should add a param to filter out links for gpt function calls, but not for the slash command where these links are useful in chat